lab/homework/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "13-npm",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha -bdd test.js"
+  },
+  "dependencies": {
```

Comment:
These are actually `devDependencies` as you use them in tests, not in the source code of your package.